mlperf_logging/rcp_checker/1.0.0/rcps_bert.json

```diff
@@ -38,42 +38,42 @@
        2508800, 2458624, 2684416, 2533888, 2533888, 2784768, 2308096, 2784768, 2584064, 2809856]
   },
 
-  "bert_ref_768":
+  "bert_ref_1536":
```

Comment:
Do you want to maybe submit a different PR for bert? As this PR becomes larger and larger, it might become more difficult to review.

Reply:
This is the first change to that file -- how does it complicated things?

Reply:
The title of this PR says "RCP update: **Resnet 64K** and **Unet3D**"

Reply:
Well yes, I opened that almost a week ago, but it takes a long time to review, and approve.

Reply:
If I open another one it will take even longer.

Reply:
Fair enough, that's okay then.